### PR TITLE
Refactor error popups for config JSON loading

### LINKED ISSUE #275: Crash when loading old config
Since this is all in development there probably isn't a guarantee that an old config will work with newer builds. _However_ it still shouldn't crash the program.

Running on Windows 10 64-bit using v2022-1.1-beta4 (btw, you should make it so I can select the text in the info box so I can paste it here).

Loading the attached config crashes the program. I don't see any logs either in AppData or on the console
[mk2_swerve_sterring_config.zip](https://github.com/wpilibsuite/sysid/files/7785719/mk2_swerve_sterring_config.zip)
.

### PR DESCRIPTION
**Pull request by Piphi5**
Fixes #275

### CODE REVIEW

File: sysid-application/src/main/native/include/sysid/Util.h [hunk 1]
```diff
@@ -46,6 +46,8 @@ static constexpr const char* kUnits[] = {"Meters",  "Feet",      "Inches",
  */
 void CreateTooltip(const char* text);
 
+void CreateErrorPopup(bool& isError, std::string_view errorMessage);
+
```
Comment (Piphi5) [inline sysid-application/src/main/native/include/sysid/Util.h:50]: This should have failed the documentation check

Comment (calcmogul) [reply sysid-application/src/main/native/include/sysid/Util.h:50]: That's because doxygen isn't processing any files. The docs it generates are blank.

Comment (calcmogul) [reply sysid-application/src/main/native/include/sysid/Util.h:50]: https://github.com/wpilibsuite/sysid/pull/283 at least generates something, but it still isn't warning on undocumented functions.

File: sysid-application/src/main/native/include/sysid/Util.h [hunk 2]
```diff
@@ -46,6 +46,8 @@ static constexpr const char* kUnits[] = {"Meters",  "Feet",      "Inches",
  */
 void CreateTooltip(const char* text);
 
+void CreateErrorPopup(bool& isError, std::string_view errorMessage);
```
Comment (prateekma) [inline sysid-application/src/main/native/include/sysid/Util.h:49]: Can you add docs to this so we can merge?

**Review by prateekma — DISMISSED** [review]

**Review by calcmogul — APPROVED** [review]

**Pull request merged**